src/ui/Windows/MainView.cpp

```diff
@@ -2558,7 +2558,11 @@ void DboxMain::OnTimer(UINT_PTR nIDEvent)
     if (prefs->GetPref(PWSprefs::UseSystemTray)) {
```

Comment:
I've negated the predicate and moved the call to ShowWindow(SW_HIDE) outside of the if statement to make it a bit clearer.